scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -136,19 +144,56 @@ public void connect() throws Exception {
      * @throws Exception
      */
     public void connect(String url) throws Exception {
+        connect(url, SchedulerNodeClient.this.schedulerRestUrl, SchedulerNodeClient.this.schedulerRestPublicUrl);
+    }
+
+    private void connect(String url, String backupUrl1, String backupUrl2) throws Exception {
```

Comment:
Why do  we need a **backupUrl2** ? _(I see it is set to null)_

Reply:
it is used when we do schedulerapi.connect(url)

```
public void connect(String url) throws Exception {
        connect(url, SchedulerNodeClient.this.schedulerRestUrl, SchedulerNodeClient.this.schedulerRestPublicUrl);
 }
```